iphone/cli/commands/_build.js

```diff
@@ -7142,6 +7142,22 @@
 	return crypto.createHash('sha1').update(value).digest('hex');
 }
 
+// This function has the advantage to detect bridges architectures
+// like x86_64 in Rosetta mode (process.arch would still print "arm64" in that case)
+async function processArchitecture() {
+    return new Promise((resolve, reject) => {
+        const exec = require('child_process').exec;
+
+        exec('uname -m', function (error, stdout, stderr) { 
+          if (error) {
+            reject(error);
+            return;
+          }
+          resolve(stdout);
+        });
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 7157;
-->
- :no_entry_sign: iphone/cli/commands/_build.js line 7157 – Expected indentation of 2 tabs but found 8 spaces. (indent)